iphone/Classes/TiUISearchBarProxy.m

```diff
@@ -184,9 +184,52 @@ - (id)barColor
 
 - (NSNumber *)style
 {
-  return NUMINT([[self searchBar] searchBarStyle]);
+  return @([[self searchBar] searchBarStyle]);
 }
 
+#if IS_SDK_IOS_13
+- (void)insertTokenAtIndex:(id)params
+{
+  ENSURE_ARG_COUNT(params, 2);
+
+  if (![TiUtils isIOSVersionOrGreater:@"13.0"]) {
+    return;
+  }
+
+  NSDictionary<NSString *, NSString *> *token = params[0];
+  int index = [TiUtils intValue:params[1]];
+
+  UISearchToken *searchToken = [UISearchToken tokenWithIcon:[TiUtils toImage:token[@"image"] proxy:self]
+                                                       text:[TiUtils stringValue:@"text" properties:token]];
+
+  if (token[@"identifier"] == nil) {
+    NSLog(@"[WARN] Missing search token identifier! Using a generated UUID …");
+  }
+
+  searchToken.representedObject = [TiUtils stringValue:@"identifier" properties:token def:[TiUtils createUUID]];
+
+  [[[self searchBar] searchTextField] insertToken:searchToken atIndex:index];
+}
+
+- (void)removeTokenAtIndex:(id)index
+{
+  ENSURE_SINGLE_ARG(index, NSNumber);
+  [[[self searchBar] searchTextField] removeTokenAtIndex:[TiUtils intValue:index]];
```

Comment:
It will crash for iOS < 13. Apply check.
```suggestion
  if (![TiUtils isIOSVersionOrGreater:@"13.0"]) {
     return;
   }
  [[[self searchBar] searchTextField] removeTokenAtIndex:[TiUtils intValue:index]];
```

Reply:
Is this really required? The docs already state that it's iOS 13+. If someone calls an unavailable method in other frameworks / languages, it crashes as well. I am trying to keep the code as clean as possible.

Reply:
I think it is required. It may be we can log a message to know developer that this api is not available rather crashing app. If I see other implementations, it has check for APIs with iOS versions. 
You have also added check in function  "- (void)insertTokenAtIndex:(id)params" :)